### ISSUE TITLE
Aqara Magic Cube

### ISSUE THREAD

**Issue by PandaUnit-TWL** [SirGoodenough/HA_Blueprints#5]
Hey. First off thanks for making the HA templates. I am using the Aqara Cube one and it's been great.
But I ran into an issue today that I don't know if it even is an issue.
I had the template working well to control a TV and so on but after some error with Zigbee2MQTT (z2m) that kept devices from reporting their state to HA, and an well timed z2n update release that fixed this, suddenly the template won't function.
I'm guessing it's the template because if I set up an automation manually everything works fine. Only that which used, and or newly uses, the Aqara Cube blueprint won't function.
I tried a new Automation based off the blueprint, and I tried deleting the blueprint and re adding it from GitHub. Neither helped.
Any ideas?

**What version of the blueprint has the issue?**
Aqara Magic Cube Zigbee2MQTT - 2022-05-05

**What was the version of Home Assistant Core?**
core-2022.7.7

**What type of installation are you running?**
Home Assistant Supervised

**YAML snippets INCLUDING the Script or Automation that you built to call the blueprint**
```
- id: '1659285890446'
  alias: Aqara Magic Cube LR
  description: ''
  use_blueprint:
    path: SirGoodenough/Zigbee2MQTT - Xiaomi Cube Controller.yaml
    input:
      remote: sensor.aqara_smart_cube_lr_action
      doubletap_any: []
      rotate_ccw_any: []
      doubletap_face_0:
      - service: media_player.media_play_pause
        data: {}
        target:
          entity_id: media_player.android_tv_192_168_1_21
      rotate_cw_face_0:
      - service: media_player.volume_up
        data: {}
        target:
          entity_id: media_player.android_tv_192_168_1_21
      - service: media_player.volume_up
        data: {}
        target:
          entity_id: media_player.android_tv_192_168_1_21
      rotate_ccw_face_0:
      - service: media_player.volume_down
        data: {}
        target:
          entity_id: media_player.android_tv_192_168_1_21
      - service: media_player.volume_down
        data: {}
        target:
          entity_id: media_player.android_tv_192_168_1_21
      slide_face_0:
      - service: media_player.volume_mute
        data:
          is_volume_muted: false
        target:
          entity_id: media_player.android_tv_192_168_1_21
      doubletap_face_3:
      - if:
        - condition: device
          device_id: 42f26eca0268c27680edb01d86860ea7
          domain: media_player
          entity_id: media_player.android_tv_192_168_1_21
          type: is_on
        then:
        - service: media_player.turn_off
          data: {}
          target:
            entity_id: media_player.android_tv_192_168_1_21
        else:
        - service: media_player.turn_on
          data: {}
          target:
            entity_id: media_player.android_tv_192_168_1_21
```

**Additional context**

**Comment by SirGoodenough** [comment]
ire the 2 legacy settings set according to the instruction document? Also
contacting in 1 place is better than me needing to respond in 3 places, so
just pick one to answer in. thanks


_|I/V\©    Typed on a phone.  Please excuse and spelling errors or
auto-correct blunders.

**Comment by PandaUnit-TWL** [comment]
I tried checking off Legacy API and availability payload then restarting z2m but it didn't work.

Then I also checked off the "Home Assistant legacy entity attributes" option under "Home Assistant Integrations" as the help says and it does work again. I've also turned off the Legacy API and availability payload and it still works.

Any idea why it originally worked and only now it needs this one of 3 or 4 settings on?
